src/js/components/DataTable/StyledDataTable.js

```diff
@@ -83,6 +83,11 @@ const StyledDataTableRow = styled(TableRow)`
     `
     cursor: pointer;
   `}
+
+  &:focus {
```

Comment:
Just wondering why we are changing the pattern and putting focus on the row instead of the body.

Reply:
As part of the trimming down I've done, this will be removed as well. I believe where we check for `props.active` that mimics the same functionality.